test/acceptance/features/steps/openshift.py

```diff
@@ -397,6 +397,14 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
 
+    def search_namespace_for_resource(self, resource_plural, name):
```

Comment:
perhaps to name the method as `lookup_namespace_for_resource`?

Reply:
+1